client/src/pages/proposalDetails/ProposalDetails.js

```diff
@@ -53,15 +49,34 @@ function amountToDollarString(amount) {
 
 function ProposalConditionalRender(isAdmin) {
 
-  var VOTE_VALUE = false;
-  var PROGRESS_VALUE = !VOTE_VALUE;
-
-  const [showProgressBars, setToggleView] = useState(VOTE_VALUE);
+  var PROGRESS_VALUE = false;
+  const [showProgressBars, setToggleView] = useState(!PROGRESS_VALUE);
```

Comment:
The convention for writing state hooks is to name them [hookName, setHookName]. Try to stick to this for code readability